packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -978,6 +1289,9 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
       bottomSheet.animationController.dispose();
     if (_currentBottomSheet != null)
       _currentBottomSheet._widget.animationController.dispose();
+    _fabMoveController.removeStatusListener(_onFabMoveStatus);
+    _fabMoveController.dispose();
```

Comment:
no need to remove the listener if you immediately dispose

Reply:
Done